flake8_class_attributes_order/checker.py

```diff
@@ -32,6 +32,12 @@ def add_options(cls, parser) -> None:
             help='Comma-separated list of class attributes to '
                  'configure order manually',
         )
+        parser.add_option(
```

Comment:
Could you please explain the idea of this change? I honestly don't understand, why to ignore docstring and why it should be default behavior.

Reply:
The motivation for skipping the docstring is an optional Python code style, where you declare fields and variables documentation under variables.

It looks like this:

```
SOME_CONSTANT = "SOME_VALUE"
"""
I want to add an explanation to this constant because IDEs like Pycharm, VSCode, or Neovim will highlight it. It is not a documentation string of the file, but it is pretty useful in some projects. 
```

See rejected https://peps.python.org/pep-0224/

It is not the default python behavior, but used by some developers.